Library/Homebrew/utils/analytics.sh

```diff
@@ -113,6 +113,8 @@ report-analytics-screenview-command() {
       "${args[@]}" \
       --silent --output /dev/null &>/dev/null & disown
   else
+    echo "$HOMEBREW_CURL" https://www.google-analytics.com/debug/collect \
+      "${args[@]}"
```

Comment:
For `echo`ing this, I think it's better to put everything in a single string (like in the Ruby code) instead of passing everything as separate arguments to `echo`:

```sh
echo "$HOMEBREW_CURL https://www.google-analytics.com/debug/collect ${args[*]}"
```